using_openshift/builds.adoc

```diff
@@ -17,7 +17,7 @@ We defined three types of builds:
 * STI build
 * custom build.
 
-More of this is described link:../architecture/builds.html[here].
+These are described in more detail link:../architecture/builds.html[here].
```

Comment:
That makes sense, and I now that I think of it, I do prefer using the topic title.